### PR TITLE
Token support (rspy-352-357)

### PR DESCRIPTION
**Pull request by agrosu1978**

### CODE REVIEW

File: charts/mockup-station-cadip/templates/configmap.yaml [hunk 1]
```diff
@@ -25,8 +25,12 @@ metadata:
 data:
   {{ .Values.app.authConfigFile }}: |-
     {
-        "test": "$2b$12$MfKwr.11KN7OjCqLLwh/yuLhXasqSzTTdsz71YlnoeZdqlELqlyQi",
-        "ovidiu": "$2b$12$KNiiNLAIbJDj/ftcVidt1.mVxSn4rttIByudz9GAx5IqPEsWg38jW"
+        "client_id": "client_id",
+        "client_secret": "client_secret",
+        "username" : "test",
```
Comment (Padeanu) [inline charts/mockup-station-cadip/templates/configmap.yaml:30]: maybe not that important, but you could use b64encrypt here, as before

File: charts/rs-server-station-secrets/values.yaml [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+      service: 
+        name: auxip
+        url: http://mockup-station-adgs-svc.processing.svc.cluster.local:8080                    
+      domain: mockup-station-adgs-svc.processing.svc.cluster.local
+      authentication:
+        auth_type: oauth2
+        token_url: http://mockup-station-adgs-svc.processing.svc.cluster.local:8080/oauth2/token
+        grant_type: password
+        scope: null
+        username: test
```
Comment (Padeanu) [inline charts/rs-server-station-secrets/values.yaml:30]: is it normal to have this values in clear, here?

Comment (agrosu1978) [reply charts/rs-server-station-secrets/values.yaml:30]: yes, they are programmatically overwritten

File: charts/mockup-station-cadip/templates/configmap.yaml (hunk 1 above)
Comment (agrosu1978) [reply charts/mockup-station-cadip/templates/configmap.yaml:30]: not that important, it's just a mockup for tests

**Review by nleconte-csgroup — APPROVED** [review]

**Pull request merged**